docs/proxy.md

```
@@ -381,6 +381,48 @@ proxy:
+### Example
+
+```yaml
+# config.yaml — global allowlist (all users)
+proxy:
+  policy:
+    allowed_models:
+      - provider: anthropic
+        models: ["claude-sonnet-4-*", "claude-opus-4-*"]
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)